app/main/views/suppliers.py

```diff
@@ -361,3 +359,66 @@ def set_selection_questions(supplier_id, framework_slug):
         abort(400, "Database Error: {}".format(e))
 
     return jsonify(selectionAnswers=supplier_framework.serialize()), status_code
+
+
+@main.route('/suppliers/<supplier_id>/frameworks/interest', methods=['GET'])
```

Comment:
Is this how this is used on the frontend? Don't we ask about interest for a specific framework?